src/lib/ui/containers/SigninForm/SigninForm.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+  import { createEventDispatcher, getContext } from "svelte";
+  import { Errors, Field, faivform } from "@tuentyfaiv/svelte-form";
+  import { getTexts } from "$lib/logic/utils/objects.js";
+  import { fieldsSignin } from "$lib/logic/schemas/index.js";
+
+  import type { Readable } from "svelte/store";
+  import type { FieldInputForm } from "$lib/logic/typing/globals/interfaces.js";
+  import type { ConfigForm } from "$lib/logic/typing/store/config.js";
+  import type { Props, SigninFields } from "./SigninForm.proptypes.js";
+
+  import * as stylesinternal from "./SigninForm.styles.js";
+
+  export let submit: Props["submit"];
+  export let context: Props["context"] = "form";
+  export let showErrors: Props["showErrors"] = true;
+  export let styles: Props["styles"] = undefined;
```

Comment:
🚫 **[eslint]** <[no-undef-init](https://eslint.org/docs/latest/rules/no-undef-init)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>It's not necessary to initialize 'styles: Props["styles"]' to undefined.
```suggestion
  export let styles: Props["styles"];
```